crates/testlang/outputs/spec/generated/public/supported-versions.md

```diff
@@ -0,0 +1,9 @@
+<!-- This file is generated automatically by infrastructure scripts. Please don't edit by hand. -->
+
+This specification compiles information from 4 publicly released versions of Solidity:
```

Comment:
Found a bug in `codegen_spec`!
I will submit a fix for it after this PR though. No reason to block on it.

```suggestion
This specification compiles information from 4 publicly released versions of Testlang:
```